modules/oe_translation_poetry/modules/oe_translation_poetry_mock/src/Controller/PoetryMockController.php

```diff
@@ -203,7 +203,13 @@ public function sendTranslationNotification(JobInterface $tmgmt_job): Response {
     $item = reset($items);
     $data = \Drupal::service('tmgmt.data')->filterTranslatable($item->getData());
     foreach ($data as $field => &$info) {
-      $info['#text'] .= ' - ' . $tmgmt_job->getTargetLangcode();
+      if (isset($info['#translation'])) {
+        $info['#text'] = $info['#translation']['#text'] . ' UPDATE';
```

Comment:
We already have a "translation update" that has nothing to do with this operation.
I suggest here ' OVERRIDDEN'.